pkgs/tools/inputmethods/fcitx5/fcitx5-lua.nix

```diff
@@ -1,13 +1,16 @@
-{ lib, stdenv
+{ pkgs
+, lib
+, stdenv
 , fetchFromGitHub
 , cmake
 , extra-cmake-modules
 , fcitx5
 , lua5_3
```

Comment:
I prefer when it's just `lua` here and overridden with `lua = lua5_3` in the callPackage call. It's more future proof etc.